src/content/docs/en/reference/api-reference.mdx

```diff
@@ -826,6 +826,200 @@ If true, the cookie is only set on https sites.
 
 Allows customizing how the cookie is serialized.
 
+### `session`
+
+<p>
+
+**Type:** `AstroSession`
+
+<Since v="5.1.0" />
```

Comment:
```suggestion
<Since v="5.7.0" />
```
We should put the first *stable* version here. And, as for the items following, I *would* tend to also mention 5.7.0 *if* you think it's likely more will be added here. If it's not super likely that this will regularly update, then fine to just keep it to the parent.